docs/pages/inboxes/push-notifs/understand-push-notifs.md

```diff
@@ -0,0 +1,141 @@
+# Understand push notifications with XMTP
+
+With XMTP, you can enable real-time push notifications to keep users updated on new conversations and messages.
+
+At the highest level, push notifications with XMTP require these three elements:
+
+1. An **XMTP push notification server** that listens for all messages sent on the XMTP network. You set the server to listen to the`production`, `dev`, or `local` environment, and every message sent using that environment flows through the server. The server filters the messages accordingly and sends only the desired push notifications to a push notification service.
+
+2. A **push notification service**, such as Apple Push Notification service (APNs), Firebase Cloud Messaging (FCM), or W3C Web Push protocol, receives push notifications from the XMTP push notification server.
```

Comment:
@rygine - correct reference to W3C Web Push protocol for web apps?

Reply:
yep!